gym/wrappers/monitoring/video_recorder.py

```diff
@@ -169,6 +180,12 @@ def functional(self):
 
     def capture_frame(self):
         """Render the given `env` and add the resulting frame to the video."""
+        frame = self.env.render(mode=self.render_mode)
```

Comment:
you should avoid calling `render` with mode argument when env.render_mode was set at the beginning because this will show a warning